list.md

```diff
@@ -85,6 +85,9 @@ Deutsche Zentralbibliothek für Medizin (ZB MED)
 UB Ilmenau
 [https://github.com/ubilmenau](https://github.com/ubilmenau)
 
+project finc
```

Comment:
Hi Stefan,

Ok , I agree. I try to find some more precise. The project is based mainly at Leipzig University Library, but the focus isn't so restricted to one institution. So I'll reflect about a better naming for a while and send a new suggestion.